src/main/java/uk/gov/companieshouse/api/testdata/model/entity/CompanyProfile.java

```diff
@@ -50,6 +197,62 @@ public void setCompanyNumber(String companyNumber) {
         this.companyNumber = companyNumber;
     }
 
+    public Instant getDateOfCreation() {
+        return dateOfCreation;
+    }
+
+    public void setDateOfCreation(Instant dateOfCreation) {
+        this.dateOfCreation = dateOfCreation;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public Boolean getUndeliverableRegisteredOfficeAddress() {
+        return undeliverableRegisteredOfficeAddress;
+    }
+
+    public void setUndeliverableRegisteredOfficeAddress(Boolean undeliverableRegisteredOfficeAddress) {
+        this.undeliverableRegisteredOfficeAddress = undeliverableRegisteredOfficeAddress;
+    }
+
+    public String getCompanyName() {
+        return companyName;
+    }
+
+    public void setCompanyName(String companyName) {
+        this.companyName = companyName;
+    }
+
+    public List<String> getSicCodes() {
+        return sicCodes;
+    }
+
+    public void setSicCodes(List<String> sicCodes) {
+        this.sicCodes = sicCodes;
+    }
+
+    public ConfirmationStatement getConfirmationStatement() {
+        return confirmationStatement;
+    }
+
+    public void setConfirmationStatement(ConfirmationStatement confirmationStatement) {
+        this.confirmationStatement = confirmationStatement;
+    }
```

Comment:
We are creating an object on creation of this one so I'd force to use the getter when we need to change its values rather than allowing callers to set a new confirmation statement object via this setter. 
Same for Accounts